dataland-frontend/src/components/resources/frameworkDataSearch/sme/DataModelsTranslations.ts

```diff
@@ -0,0 +1,11 @@
+export const detailsCompanyDataTableColumnHeaders = {
```

Comment:
The file name and const name seem unfitting.

Reply:
I went with what we already had for lksg where the framework was also only specified in the folder name. However I was also wondering about that and I think I should change this. So no confusion will be caused between the lksg and sme version of this mapping.